src/modules/user_choice.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../../app'
+
+def menu
+  puts "\n Welcome to School Library App! \n\n"
+  puts "Please chose an option by entering a number: \n\n"
+  options = {
+    1 => 'list_of_books',
+    2 => 'list_of_people',
+    3 => 'create_a_person',
+    4 => 'create_book',
+    5 => 'create_rental',
+    6 => 'list_all_rentals_for_person',
+    7 => 'Exit'
+  }
+  puts options.map { |key, value| "#{key}. #{value}" }.join("\n")
+end
+
+def option_case(choice)
+  case choice
+  when 1
+    load_books
+  when 2
+    load_people
+  when 3
+    create_person
+  when 4
+    create_book
+  when 5
+    create_rental
+  when 6
+    load_rentals
+  else
+    puts '*** Incorect choice, chose a number between 1-7 ***'
+  end
+end
```

Comment:
- [ ] Great job for extracting some pieces of logic into separate class(es). but I think you missed the first point to focus on keep the functionality for the intial choices in main.rb file since you made it in user_choice file but it would be great it you add add it in main.rb file not here as required ✨💯🤩


![image](https://user-images.githubusercontent.com/58553711/224853016-178d7beb-73f8-4c5c-bdd7-56044be55fd6.png)

Reply:
Good day, @Reem-lab I didn't comprehend what you meant by the above statement, and I believe that the only purpose of our "main.rb" file is to start and exit the program.
```sh
def main
  app = App.new
  app.run
end
main
```
Thank you.